lib/services/pubsub/service.js

```diff
@@ -771,7 +794,7 @@ PubSubService.prototype.publish = function (message, payload) {
   opts.hasWildcard = true; // all remaining emit attempts are to wildcard subscribers
 
   for (var allPath in this.__listeners_wildcard_ALL) {
-    if (this.happn.utils.wildcardMatch(allPath.replace('/ALL@/', '/*@/'), messageChannel)) {
+    if (this.happn.utils.wildcardMatch(allPath.replace('/ALL@', '/*@'), messageChannel)) {
```

Comment:
@southbite , please could you either do this optimization, or create a ticket if it's not opportune right now